TestApp/app/screens/DataScreen.js

```diff
@@ -103,10 +103,10 @@ export default class DataScreen extends Component {
               this.setState({ docTtl: itemValue })
             }
           >
-            <Picker.Item label="Default" value="-1" />
-            <Picker.Item label="No Cache" value="0" />
-            <Picker.Item label="2 seconds" value="2" />
-            <Picker.Item label="Infinite" value="-1" />
+            <Picker.Item label="Default" value={-1} />
```

Comment:
Please apply the same fix for DemoApp whenever applicable so that we don't forget DemoApp.